php/classes/ajax/class-ajax-handler.php

```diff
@@ -198,19 +156,13 @@ public function update_episode_embed_code() {
 	 * Import an external RSS feed via ajax
 	 */
 	public function import_external_rss_feed() {
-		// @todo add nonces, add user caps check, validate inputs
+		self::import_security_check();
```

Comment:
To stick with the "development style" of the plugin, can this be replaced with 
```
$this->import_security_check();
```

Reply:
self:: is usually only needed with static functions.

Reply:
Ah, good point.